include/nbl/video/IPhysicalDevice.h

```diff
@@ -126,17 +225,98 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
                 return static_cast<uint32_t>(core::min<uint64_t>(infinitelyWideDeviceWGCount,maxResidentWorkgroups));
             }
         };
-        const SLimits& getLimits() const { return m_limits; }
+        
+        struct SProperties
+        {
+            //--> VkPhysicalDeviceProperties:
+            APIVersion  apiVersion;
+            // uint32_t driverVersion;
+            // uint32_t vendorID;
+            // uint32_t deviceID;
+            E_TYPE      deviceType;
+            // char     deviceName[VK_MAX_PHYSICAL_DEVICE_NAME_SIZE];
+            // uint8_t  pipelineCacheUUID[VK_UUID_SIZE];
+            SLimits     limits;
+            // VkPhysicalDeviceSparseProperties    sparseProperties;
+
+            //--> VkPhysicalDeviceDriverProperties
+            E_DRIVER_ID driverID;
+            // char driverName[VK_MAX_DRIVER_NAME_SIZE];
+            // char driverInfo[VK_MAX_DRIVER_INFO_SIZE];
+            // VkConformanceVersion conformanceVersion;
```

Comment:
Conformance in OpenGL:
INTEL = 4.4
NVIDIA = 4.4
AMD = 3.3

I guess `driverInfo` can be `GL_RENDERER` again, but `driverName` could just be the `E_DRIVER_ID` enum but as a string